source/_docs/installation/macos.markdown

```diff
@@ -11,12 +11,15 @@ footer: true
 
 [macOS](http://www.apple.com/macos/) is available by default on Apple computer. If you run a different operating system, please refer to the other section of the documentation.
 
-To run Home Assistant on macOS you need to install Python first. Download Python 3.5.3 or later from [https://www.python.org/downloads/mac-osx/](https://www.python.org/downloads/mac-osx/) and follow the instructions of the installer.
+To run Home Assistant on macOS you need to install Python first. Download Python 3.6 or later (at this time we recommend Python 3.7) from [https://www.python.org/downloads/mac-osx/](https://www.python.org/downloads/mac-osx/) and follow the instructions of the installer.
```

Comment:
Maybe we should switch to `brew` for this? Dunno... ?

Reply:
If you can provide a pointer to how - I've not touched a Mac in more years than I care to count

Reply:
Left it out for now. Since it would require details on how to setup Brew as well.